### ISSUE TITLE
train bytetrack_s_mot17 issue

### ISSUE THREAD

**Issue by Dawn-JN** [ifzhang/ByteTrack#112]
Hello! I tried to train bytetrack_s_mot17(YOLOXs) twice, and both time the AP is 0.593(MOT17 Train), however AP=0.639 when using the provided weight.I train and test like this (with 8 K80 GPUs):
`python tools/train.py -f exps/example/mot/yolox_s_mix_det.py -d 8 -b 48 --fp16 -o -c pretrained/yolox_s.pth
python tools/track.py -f exps/example/mot/yolox_s_mix_det.py -c pretrained/trained.pth.tar -b 1 -d 1 --fp16 --fuse`
Is there anything different or needs to be changed when training bytetrack_s_mot17(YOLOXs)?  Please help me, thanks a lot!

**Comment by Dawn-JN** [comment]
I notice that in paper "...with batch size of 48....The initial learning rate is 0.001",but in yolox_s_mix_det.py, line 29
`self.basic_lr_per_img = 0.001 / 64.0`
The learning rate here is divided by 64, it's the same as YOLOX, but YOLOX's batchesize is 64 and bytetrack's is 48. so should it be replaced by 48 here?

**Comment by Dawn-JN** [comment]
I notice that in paper "...with batch size of 48....The initial learning rate is 0.001",but in yolox_s_mix_det.py, line 29
`self.basic_lr_per_img = 0.001 / 64.0`
The learning rate here is divided by 64, it's the same as YOLOX, but YOLOX's batchesize is 64 and bytetrack's is 48. 
And in before_train function(trainer.py, line 154),
`self.lr_scheduler = self.exp.get_lr_scheduler(
            self.exp.basic_lr_per_img * self.args.batch_size, self.max_iter
        )`
So should it be replaced by 48 here?
